header.php

```diff
@@ -57,5 +57,51 @@
 
   <!-- start content -->
   <header id="header">
+  <?php
+    $logo = IGV_get_option('_igv_metadata_logo');
```

Comment:
could possibly get the `_igv_metadata_logo_id` and then use a custom size for the logo here